SimulationAgent/appsettings.ini

```diff
@@ -4,6 +4,10 @@ iothub_connstring = "${PCS_IOTHUB_CONNSTRING}"
 # which can be mounted to inject custom device models and scripts.
 device_models_folder = ./data/devicemodels/
 device_models_scripts_folder = ./data/devicemodels/scripts/
+; Disable twin read/writes and methods, to reduce the number of threads
+; and open connections.
+twin_read_write_enabled = false;
```

Comment:
I would use an optional Env Var here:
1. if the env var is empty, default to True (RMv2 behavior)
2. if the env var is available, it can be any of T,F,true,false,on,off,0,1,-1 (same as the implementation of "auth_required" in the webservice)

Reply:
If we default to true, then the service will have twin read/write enabled by default. Don't we want it to default to false for performance reasons or is this something the CLI will set on deployment?

Reply:
Update: This needs to be backwards compatible with RMv2, so we should default to true and the CLI will set the config value of false.